tools/cryptogenotyper/cryptogenotyper.xml

```diff
@@ -0,0 +1,141 @@
+<tool id="CryptoGenotyper" name = "CryptoGenotyper" version="@VERSION@">
+  <description>
+    CryptoGenotyper is a standalone tool to analyze Cryptosporidium Sanger sequencing data and classify the species and subtype based on SSU rRNA and gp60 gene markers, respectively.
```

Comment:
This description will be in the tool panel, it seems a little bit too long.

Maybe something like "classify species and subtypes based on SSU rRNA and gp60 on Cryptosporidium Sanger sequencing data"?

Reply:
Changed to a shorter title to make it look prettier.